packages/twenty-front/src/modules/object-record/record-inline-cell/components/RecordInlineCell.tsx

```diff
@@ -83,17 +102,11 @@ export const RecordInlineCell = ({ readonly }: RecordInlineCellProps) => {
       labelWidth={fieldDefinition.labelWidth}
       showLabel={fieldDefinition.showLabel}
       editModeContent={
-        <FieldInput
-          recordFieldInputdId={`${entityId}-${fieldDefinition?.metadata?.fieldName}`}
-          onEnter={handleEnter}
-          onCancel={handleCancel}
-          onEscape={handleEscape}
-          onSubmit={handleSubmit}
-          onTab={handleTab}
-          onShiftTab={handleShiftTab}
-          onClickOutside={handleClickOutside}
-          isReadOnly={readonly}
-        />
+        fieldDefinition.type === 'DATE_TIME' ? (
```

Comment:
we should either handle all fields the same way or either compute it through an isAbsolutePositionnedField helper?